docs/docs/pricing/README.md

```diff
@@ -149,39 +162,37 @@ For example, if you sign up on Jan 1st, your first billing period will last one
 
-### Base Invocations Quota
+### Included Credits
 
-When you sign up for a paid plan, you pay a platform fee at the start of each [billing period](#billing-period). This minimum monthly charge grants you a base of invocations that you can use for the rest of your billing period (see your [Billing and Usage Settings](https://pipedream.com/settings/billing) for your exact quota). If you have been granted any additional invocation increases by Pipedream, that is added to the base quota. **This total is called the base invocations quota**.
+When you sign up for a paid plan, you pay a platform fee at the start of each [billing period](#billing-period). This minimum monthly charge grants you a base of included credits that you can use for the rest of your billing period (see your [Billing and Usage Settings](https://pipedream.com/settings/billing) for your exact quota). If you have been granted any additional credit increases by Pipedream, that is added to the included credits.
 
-### Additional Billable Invocations
+### Additional Credits
 
-Any invocations you run over your [base invocations quota](#base-invocations-quota) are called **additional billable invocations**. This usage is added to the invoice for your next [billing period](#billing-period), according to the [invoicing cycle described here](#when-am-i-invoiced-billed-for-paid-plans).
+Any credits you run over your [included credit](#included-credits) are called **additional credits**. This usage is added to the invoice for your next [billing period](#billing-period), according to the [invoicing cycle described here](#when-am-i-invoiced-billed-for-paid-plans).
 
-## FAQ
-
-### What are invocations?
+### Data Store Keys
 
-Pipedream counts an **invocation** each time a workflow or event source is triggered by an incoming event.
+A Data Store key represents a single record in a Data Store.
 
-Pipedream increments the count of invocations by one for each incoming event, regardless of the number of steps in your workflow. For example, if you send an HTTP request to a workflow with five steps, Pipedream will count that as one invocation.
+In the example below, there are two records in the Data Store, and therefore there are two keys total.
 
-Moreover, if you have a workflow triggered by a cron job running once a minute for the entire day, that will incur 1,440 invocations (60 minute \* 24 hours). For cron-triggered workflows or event sources, remember that you can always reduce the frequency to reduce your invocations.
+![Example of a Data Store with two keys](https://res.cloudinary.com/pipedreamin/image/upload/v1673537163/docs/CleanShot_2023-01-12_at_10.25.25_z6yg8t.png)
 
-If an event emitted by an event source triggers a single workflow, that will count as **two** invocations: one for the source, and one for the workflow. In other words, source and workflow execution is distinct: each counts invocations on its own.
+## FAQ
```

Comment:
I think all of these may be good to include on the FAQ at https://pipedream.com/pricing once that launches to production.